include/bluetooth/mesh/light_ctrl_srv.rst

```diff
@@ -0,0 +1,371 @@
+Note that the transition time between states in the Light LC Server makes the feedback loop more forgiving, and a larger transition time can compensate for poor regulator response.
+
+The illuminance regulator is a PI regulator, which uses the instantaneous error (the proportional component) and the accumulated error (the integral component) to compensate for a mismatch between the reference and the measured level.
+
+The proportional component is typically the main source of correction for a PI regulator. It compares the reference value to the most recent feedback value, and naively attempts to correct the error in every step.
+
+The integral component compensates for errors by adding up the sum of the error over time.
+Its main contribution is to eliminate system bias and accelerate the system step response.
+
+Changing different coefficients will affect the step response differently.
+Increasing the two coefficients will have the following effect on the step response:
+
+=========== ========= ========= ============= ==================
+Coefficient Rise time Overshoot Settling time Steady-state error
+=========== ========= ========= ============= ==================
+K :sub:`p`  Faster    Higher    \-            \*
+K :sub:`i`  Faster    Higher    Longer        Reduced
+=========== ========= ========= ============= ==================
+
+The value of the coefficients is typically a trade-off between fast response time and system instability.
+Too high, and the system might become unstable, potentially leading to oscillation and loss of control.
+Too low, and the step response might be too slow or unable to reach the target value altogether.
+
+States
+======
+
+Not to be confused with the state machine states, the Light LC Server's states represent its current mode of operation and configuration.
+
+**Mode**: ``bool``
+
+Enables or disables the Light LC Server.
+When disabled, the controlled Lightness Server operates independently.
+
+**Occupancy mode**: ``bool``
+
+The occupancy mode controls whether sensor activity can turn the lights on. If disabled, motion and occupancy sensor messages may only prevent the lights from turning off, and a light switch is required to turn them on.
+
+**Light OnOff**: ``bool``
+
+When set, the Light OnOff state may trigger transactions in the lightness state machine. When read, the Light OnOff state indicates whether the lights are turned off (in Standby state) or on (in On or Prolong state).
+
+**Properties**
+
+The Light Control Properties are used to configure the Light LC Server behavior.
+See :cpp:type:`bt_mesh_light_ctrl_prop` for a list of supported properties and their representation.
+
+Extended models
+===============
+
+The Light LC Server extends the following models:
+
+- :ref:`bt_mesh_onoff_srv_readme`
+
+Additionally it requires a :ref:`bt_mesh_lightness_srv_readme` in a different element. See the :ref:`bt_mesh_light_ctrl_srv_composition` section for details.
```

Comment:
As per template, it would be great to describe here how the models relate to each other.